src/plugins/mutualGroupDMs/index.tsx

```diff
@@ -42,14 +42,14 @@ function getGroupDMName(channel: Channel) {
 export default definePlugin({
     name: "MutualGroupDMs",
     description: "Shows mutual group dms in profiles",
-    authors: [Devs.amia],
+    authors: [Devs.amia, Devs.Scab],
```

Comment:
Please don't add yourself to authors for such minor changes

```suggestion
    authors: [Devs.amia],
```

Reply:
ok